terraform/aws/analytical-platform-data-production/joiners-movers-leavers/lambda-functions.tf

```diff
@@ -7,12 +7,12 @@ module "jml_extract_lambda" {
   publish        = true
   create_package = false
 
-  function_name = "data_platform_jml_extract"
+  function_name = "analytical_platform_jml_extract"
```

Comment:
should we kebab case this? for consistency?

Reply:
Yeah I prefer that, will update now.